singlefactorauth/src/main/java/com/web3auth/singlefactorauth/SingleFactorAuth.kt

```diff
@@ -49,26 +50,24 @@ class SingleFactorAuth(singleFactorAuthArgs: SingleFactorAuthArgs) {
         loginParams: LoginParams,
         context: Context? = null,
         sessionTime: Long = 86400
-    ): CompletableFuture<TorusKey> {
-        val torusKeyCompletableFuture: CompletableFuture<TorusKey> =
-            CompletableFuture<TorusKey>()
+    ): CompletableFuture<SFAKey> {
```

Comment:
CompletableFuture<SFAKey> can be simplified to SFAKey.

You're already waiting for the result so this future is unnecessary and can be simplified into:
```
val sessionResponse: String =
                sessionManager.authorizeSession(false).get()
// then process the result
```